src/databricks_ai_bridge/external_tool_request.py

```diff
@@ -0,0 +1,57 @@
+import json as js
+from typing import Any, Dict, Optional
+
+import requests
+from databricks.sdk import WorkspaceClient
+
+from databricks_ai_bridge.utils.annotations import experimental
+
+
+@experimental
+def http_request(
```

Comment:
We should think about the module and API name a little more, maybe something like `databricks_ai_bridge.external_tools.http_request`? Or even `databricks_ai_bridge.external_tools.request`? Just thinking about how this looks in code where you might mix this with generic `requests.request` calls to some public APIs

Reply:
Another option is `databricks_ai_bridge.uc_connections.http_request` - I know we wanna present our product as external tools but what we're actually using is the UC connection to make an HTTP request in the process of defining a tool, ratehr than using a tool itself

Reply:
I have added this to `__init__.py` so for the customers what I was thinking it would be
`from databricks_ai_bridge import http_request`
What do you think? Do we need to include external_tools or un_connections here?

Reply:
Spoke offline. http_request is generic so adding the product name here would help disambiguate this. 

So changed it to `databricks_ai_bridge.external_tools.http_request`